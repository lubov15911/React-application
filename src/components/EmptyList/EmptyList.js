// @flow

import React from 'react';

import './EmptyList.scss';

const EmptyList = () => (<span className="empty-list-title">No films found</span>);

export default EmptyList;
