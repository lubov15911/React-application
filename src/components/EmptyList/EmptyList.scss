.empty-list-title {
  font-size: 5.2em;
  opacity: 0.8;
}
