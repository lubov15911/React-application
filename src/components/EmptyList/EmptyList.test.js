import React from 'react';
import { shallow } from 'enzyme';

import EmptyList from './EmptyList';

describe('EmptyList', () => {
    it('should render correctly with no props', () => {
        const component = shallow(<EmptyList />);

        expect(component).toMatchSnapshot();
    });
});
