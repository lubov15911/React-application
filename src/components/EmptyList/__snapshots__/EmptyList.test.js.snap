// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyList should render correctly with no props 1`] = `
<span
  className="empty-list-title"
>
  No films found
</span>
`;
